### ISSUE TITLE
How they designed the Equal Earth world map

### ISSUE THREAD

**Issue by kheptagon** [leios/SoME_Topics#122]
## About the author

I'm a senior math undergrad at UMD. Though I don't have a degree yet, the material would be mostly sourced from academic papers I've read about the subject.

## Quick Summary

The lesson would be about how the [Equal Earth map projection](https://en.wikipedia.org/wiki/Equal_Earth_projection) (Šavrič et al 2018) was developed, and why they made the choices they did, from a mathematical perspective. The development involved some really clever tricks that would make a good visualization.

## Target medium

A video with animated visualizations of the projections, something that Manim could achieve.

### More details

The creators set out to make a new equal-area map, starting by establishing aesthetic criteria. They used a special process to hybridize two existing equal-area projections in a way that was still equal-area. Then, to make the projection easier to implement, they made a least-squares polynomial approximation based on latitude. The latitude is re-parameterized before making the approximation to avoid instability.
The method for hybridizing equal-area projections is very interesting, and an animation would be a good way to explain and visualize it. I would cover that as well as the math behind equal-area projections and why the creators of Equal Earth made the specific mathematical choices they did. Subjects covered could include partial derivatives, the Jacobian determinant, and least-squares polynomial approximation. It's possible I could cover it with only single-variable calculus so there'd be less required knowledge.

### Contact details

I'm in the SoME discord, HEHEHE I AM A SUPAHSTAR SAGA#1163

### Additional context

Relevant academic sources:
[The Equal Earth map projection](https://www.researchgate.net/publication/326879978_The_Equal_Earth_map_projection)
[An efficient technique for creating a continuum of equal-area map projections](https://www.tandfonline.com/doi/full/10.1080/15230406.2017.1405285)

**Comment by univalency** [comment]
Do you think this kind of animation could be of help to you ? https://3b1b.github.io/manim/getting_started/example_scenes.html#surfaceexample

**Comment by kheptagon** [comment]
Yeah, that would be good for showing the Strebe method for hybridizing equal-area projections, which involves mapping between the sphere and the plane multiple times.
